### ISSUE TITLE
Why are we limited to seconds for refresh timeout?

### ISSUE THREAD

**Issue by StephenCarboni** [jwlodek/py_cui#105]
I set it to less so I could see updates from threads faster and my computer did not explode. What potential issues are there?

**Comment by StephenCarboni** [comment]
Well, I notice the CPU usage is at 1% for 100ms and 2% for 50ms. Would it be feasible to give use a redraw/inform_change method so we can tell the library that state has changed? Sorry in advance if I don't know what I'm talking about.

**Comment by jwlodek** [comment]
No particular reason I went with seconds other than I found seconds to be more intuitive. You should be able to pass in fractional values to get faster refreshes. As for the second question, you could try calling the `_draw_widgets()` function yourself on your `PyCUI` object instance, though I haven't tested anything like that myself. Ive thought about allowing something like this in a simpler way, but originally the library wasn't written in a way that it is trivial to integrate.

Basically I'd have to do some work to allow for breaking off the main draw thread and separate it from the key handling thread.
